pkg/flare/service_windows.go

```diff
@@ -296,33 +296,39 @@ func getDDServices(manager *mgr.Mgr) ([]serviceInfo, error) {
 		log.Warnf("Error getting list of running services %v", err)
 		return nil, err
 	}
+	list = filterDatadogServices(list)
+	// need to add the drivers manually as they are not returned by ListServices
+	drivers := []string{"ddnpm", "ddprocmon"}
+	list = append(list, drivers...)
 
 	for _, serviceName := range list {
-		if strings.HasPrefix(serviceName, "datadog") {
-			srvc, err := winutil.OpenService(manager, serviceName, windows.GENERIC_READ)
+		srvc, err := winutil.OpenService(manager, serviceName, windows.GENERIC_READ)
+		if err != nil {
+			log.Warnf("Error Opening Service %s: %v", serviceName, err)
+		} else {
+			conf2, err := getServiceInfo(srvc)
 			if err != nil {
-				log.Warnf("Error Opening Service %s: %v", serviceName, err)
-			} else {
-				conf2, err := getServiceInfo(srvc)
-				if err != nil {
-					log.Warnf("Error getting info for %s: %v", serviceName, err)
-				}
-				ddServices = append(ddServices, conf2)
+				log.Warnf("Error getting info for %s: %v", serviceName, err)
 			}
+			ddServices = append(ddServices, conf2)
 		}
+		srvc.Close()
```

Comment:
should not be srvc.Close() move inside non-err scrope?

Reply:
yes, good catch, thank you